pages/authority-files.xml

```diff
@@ -16,21 +16,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <sourceDesc>
                 <p>Encoded in TEI.</p>
             </sourceDesc>
-        </fileDesc>       <revisionDesc>       <change who="PL" when="2018-04-30">first version of guidelines from Wiki</change>     </revisionDesc>
+        </fileDesc>       <revisionDesc>       <change who="PL" when="2018-04-30">first version of guidelines from Wiki</change>
+            <change who="DR" when="2019-06-24">Added examples and developed</change></revisionDesc>
     </teiHeader>
     <text>
         <body xml:id="authority-files">
             <head>Authority files</head>
             <p>To each <gi>category</gi> in the <ref target="taxonomy">taxonomy</ref> corresponds a file in the authority-files repository.</p>
-            <p>In the Authority-Files repo there are currently a series of files
-                mainly generated from lists which have nothing but a title and an ID.</p>
-            <p>Each of these refers to one value so that they can be enriched with
-                a description which is out of the schema and they can be used for
-                relations and <gi>ref</gi> as the other IDs.</p>
-            <p>When reordering the hierarchy of the list, please remember to use
-                the same name you give in the taxonomy.xml file also for the filename
+            <p>In the Authority-Files repository, there are currently a series of files
+                mainly generated from lists which have nothing but a title and an ID. Each of them refers to one value.</p>
+            <p>For example, the <ref target="taxonomy">taxonomy</ref> contains the following category of keywords describing persons:</p>
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                <category>
+                    <desc>Person Types</desc>
+                    <category>
+                        <catDesc>saint</catDesc>
+                    </category>
+                    <category>
+                        <catDesc>biblical</catDesc>
+                    </category>
+                    <category>
+                        <catDesc>prophet</catDesc>
+                    </category>
+                    <category>
+                        <catDesc>angel</catDesc>
+                    </category>
+                    <category>
+                        <catDesc>martyr</catDesc>
+                    </category>
+                </category>
+            </egXML>
+            <p>For each of the values listed in children <gi>category</gi>s of the first "Person Types" <gi>category</gi>, there should be therefore a corresponding record in the
+                Authority-Files repository, see for example <ref type="bm" target="saint">saint</ref>.</p>
+            <p>Authority files records can be quite minimal, adding to the record only a title, such as in <ref type="bm" target="saint">saint</ref>:</p>
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                <titleStmt>
```

Comment:
I do not find this example terribly useful, and could in the long run cause a misunderstanding by which people create a file which contain only what is here. I would remove it, the link already provided gives more correct information.

Reply:
Sure, we can remove it